x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/formatted_field_udt.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { EuiDataGridCellValueElementProps } from '@elastic/eui';
+import React from 'react';
+
+import type { ColumnHeaderOptions, TimelineItem } from '@kbn/timelines-plugin/common';
+import type { DataTableRecord } from '@kbn/discover-utils/types';
+import { DefaultCellRenderer } from '../../cell_rendering/default_cell_renderer';
+
+export const getFormattedFields = ({
+  dataTableRows,
+  headers,
+  scopeId,
+}: {
+  dataTableRows: Array<DataTableRecord & TimelineItem>;
+  headers: ColumnHeaderOptions[];
+  scopeId: string;
+}) => {
+  return headers
+    .map((h) => h.id)
```

Comment:
Could you just get the ID in the reduce and skip the map here?

Reply:
done. cf159a76c519125e688742d8e6fd973788f47d8d.

Good catch 👌